config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net

```diff
@@ -18,7 +18,5 @@ trap clean_up EXIT SIGHUP SIGINT SIGTERM
 
 pushd docs/source/examples/workspaces/${PROVIDER}
 
-linchpin -w . -vvvv up "${TARGET}"
+linchpin -w . -v up "${TARGET}"
 
-# cleanup the network
-linchpin -w . -vvvv destroy "${TARGET}"
```

Comment:
Same as above

Reply:
@14rcole : I observed that the cleanup is happening twice when i checked the logs that's why i removed the cleanup step.